rioting-daisies/BangazonSite
# account view and edit

# Description

ticket 17 stuff



Fixes Issue # (link issue ticket here)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

fetch
checkout at-account-view
log in to an account
click on the nav bar where it says "hello admin@admin.com"
this will take you to your account view
edit some stuff, save, see the changes



# Checklist:
